product_brand/README.rst

```diff
@@ -123,12 +124,14 @@ Contributors
 -  `Trobz <https://trobz.com>`__:
 
    -  Thao Le <thaolt@trobz.com>
+   -  Do Anh Duy <duyda@trobz.com>
 
 Other credits
 -------------
 
 The migration of this module from 16.0 to 17.0 was financially supported
```

Comment:
You can remove the first sentence regarding 16.0 to 17.0 migration, and keep only the new one.